en/components/icon-service.md

```diff
@@ -0,0 +1,231 @@
+<igx-icon family="fa-solid" name="car"></igx-icon>
+```
+
+You might have noticed that for the `material` family we use the ligature `name` as name, while in the case of the `fa-solid` family we specify the `className` for name, which is `fa-car` but drop the `fa-` prefix as it has been specified when we registered the icon family in the previous step.
+
+### Adding SVG Icons
+The Ignite UI for Angular Icon Service allows us to associate SVG images with families and give them names so that they can be included via the `igx-icon` component in the same way as font-based icons. The SVGs should be resolved via either strings or via absolute URI to the SVG asset.
+
+```ts
+// Add a new 'material' icon called 'my-icon' from an SVG file
+iconService.addSvgIcon('my-icon', 'my-icon.svg', 'material');
+
+// Add a new 'fa-solid' icon called 'my-icon' from string 
+iconService.addSvgIconFromText('my-icon', '<svg>...</svg>', 'fa-solid');
+```
+
+Later in markup:
+
+```html
+<igx-icon family="material" name="my-icon"></igx-icon>
+<igx-icon family="fa-solid" name="my-icon"></igx-icon>
+```
+
+Note that we are adding custom SVG icons to families of type `liga` and `font`. This is possible because the `addSvgIcon` and `addSvgIconFromText` methods register the icons as `svg` type implicitly, allowing the `igx-icon` component to correctly render the SVG.
+
+## Meta Families
+The Ignite UI for Angular Icon Service allows us to create pseudo family maps that combine icons added via either setting a family of font icons or adding SVGs under a common umbrella so that it's easier to reference them.
+
+```ts
+// The `setIconRef` sets an icon reference in the icon map,
+// assuming material and fa-solid have been added as families,
+iconService.setIconRef('home', 'my-family', { family: 'material', name: 'home' });
+iconService.setIconRef('home-alt', 'my-family', { family: 'fa-solid', name: 'home' });
+iconService.setIconRef('car', 'my-family', { family: 'fa-solid', name: 'car' });
+```
+
+Later in markup:
+
+```html
+<igx-icon family="my-family" name="home"></igx-icon>
+<igx-icon family="my-family" name="home-alt"></igx-icon>
+<igx-icon family="my-family" name="car"></igx-icon>
+```
+
+## Icon Retrieval
+It is possible to get an icon for a given `family` and icon `name` in order to read the original icon `type`, `name`, `family`, and `className`.
+
+```ts
+const { family, className, name, type } = iconService.getIcon('my-family', 'car');
+
+console.log(family); // -> 'fa-solid'
+console.log(className); // -> 'fas'
+console.log(name); // -> 'fa-car'
+console.log(type); // -> 'font'
+```
+
+## Internal Usage
+
+Starting with version 18.1.0 of Ignite UI for Angular, we added a new `setFamily` method that allows us to create a new family of icons in the `iconService` and associate them with a CSS class.
+
+| Icon Family | Icon Name      | Target Family | Target Name          | Used By                                                 | Theme                     |
```

Comment:
At this point we can just drop the first column since it's always default, right?
And potentially rename the entire section to something to do with the default collection or explain that with version 18.1.0 all components internally use references from the `"default"` family.